master/getting-started/kubernetes/installation/hosted/calico.yaml

```diff
@@ -34,6 +34,17 @@ data:
         }
     }
 
+  # The default IP Pool to be created for each node.
```

Comment:
Probably also want to say something like "Pod IP addresses will be assigned from this pool"